playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -233,30 +249,59 @@ private MediaSource buildExoMediaSource(PKMediaSourceConfig sourceConfig) {
         if (mediaDataSourceFactory == null) {
             mediaDataSourceFactory = buildDataSourceFactory();
         }
+
+        if (subtitlesList != null) {
+            for (int subtitlePosition = 0 ; subtitlePosition < subtitlesList.size() ; subtitlePosition ++) {
+                mediaSources[subtitlePosition + 1] = buildSubtitleSource(subtitlesList.get(subtitlePosition));
+            }
+        }
+
         switch (format) {
 
             case dash:
                 if (manifestDataSourceFactory == null) {
                     manifestDataSourceFactory = buildDataSourceFactory();
                 }
-                return new DashMediaSource.Factory(
+                mediaSources[0] = new DashMediaSource.Factory(
```

Comment:
what is the meanning of ps 0 compare to pos 1 for video url?

Reply:
@GouravSna  I think 
we face here first time the need to use in window in the seek
please check this
since we always use window 0 since we did not have multiple sources before